From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Tue, 12 Jul 2016 14:26:22 +0200
Subject: [PATCH] FIXME: ODL wget: --no-check-certificate for cache.

ODL tarball is normally fetched from nexus.opendaylight.org domain,
which has a very limited bandwidth, taking up to 10-15mins for our
build server to download the ODL archive at each ISO build.
This adds up to 15-25% of each ISO build time spent on downloading
ODL archive.
Therefore, we chose to set up a local HTTPS cache of nexus repos,
(just for the sake of not hardcoding the URL in our Makefile),
and just disable certificate check for wget from local URL.

Allow using a HTTPS local cache, so we don't override the whole
URL of the ODL archive. Our local cache has HTTPS enabled, but the
certificate won't match the domain name.

FIXME: This patch is only needed if you have a local cache of
nexus.opendaylight.org!

Only use this option if you are otherwise convinced of the site's
authenticity, or if you really don't care about the validity of
its certificate.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 pre_build_hook | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pre_build_hook b/pre_build_hook
index 4785917..c3dd664 100755
--- a/pre_build_hook
+++ b/pre_build_hook
@@ -64,7 +64,7 @@ function cleanup {
 }
 
 function download {
-  wget "$1" -qO $2
+  wget --no-check-certificate "$1" -qO $2
 }
 
 function unpack {
